### PR TITLE
Migrate to function component

### PR DESCRIPTION
**Pull request by soolaymahn**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,216 +1,274 @@
...
-import React, { Component } from 'react';
+import { Tabs, Tab } from 'react-bootstrap';
+import dCheque from '../abis/dCheque.json';
+import React, { Component, useCallback, useState, useEffect } from 'react';
 import dcheque from '../dcheque.png';
 import { ethers } from 'ethers';
 import './App.css';
 
-class App extends Component {
+export const App = () => {
```
Comment (soolaymahn) [inline src/components/App.js:8]: switch to function keyword

File: src/components/App.js [hunk 2]
```diff
@@ -1,216 +1,274 @@
...
-      account: '',
-      dcheque: null,
-      dChequeAddress: null,
-      dChequeBalance: 0,
-      userBalance: 0
-    }
-  }
+  const updateReviewer = useCallback(() => {}, []);
+
+  useEffect(() => {
```
Comment (soolaymahn) [inline src/components/App.js:75]: replaces `componentWillMount`

File: src/components/App.js [hunk 3]
```diff
@@ -1,216 +1,274 @@
...
+    loadBlockchainData();
+  }, [loadBlockchainData]);
```
Comment (soolaymahn) [inline src/components/App.js:77]: remove dependency?

Comment (0xrafi) [reply src/components/App.js:77]: by removing the dependency do you mean moving this into another file? I think we def should

**Pull request merged**